packages/twenty-front/src/modules/activities/blocks/spec.ts

```diff
@@ -0,0 +1,8 @@
+import { defaultBlockSpecs } from '@blocknote/core';
+
+import { FileBlock } from './FileBlock';
+
+export const blockSpecs: any = {
```

Comment:
Is it possible to have a type here ?

Reply:
Im not sure about the types on this one - it's kind of weird what is expected by the Blockeditor generics. Maybe you or someone else has an idea how to do this the right way :)